pkg/controller/certapi/certapi.go

```diff
@@ -57,20 +71,75 @@ func New(ctx context.Context, config *config.APIServerConfig, db *database.Datab
 		return nil, fmt.Errorf("cannot create signer cache, likely invalid duration: %w", err)
 	}
 
+	mTokenExpired := stats.Int64(MetricPrefix+"/token_expired", "expired tokens on certificate issue", stats.UnitDimensionless)
+	if err := view.Register(&view.View{
+		Name:        MetricPrefix + "/token_expired_count",
+		Measure:     mTokenExpired,
+		Description: "The count of expired tokens on certificate issue",
+		TagKeys:     []tag.Key{observability.RealmTagKey},
+		Aggregation: view.Count(),
+	}); err != nil {
+		return nil, fmt.Errorf("stat view registration failure: %w", err)
+	}
+	mTokenUsed := stats.Int64(MetricPrefix+"/token_used", "already used tokens on certificate issue", stats.UnitDimensionless)
+	if err := view.Register(&view.View{
+		Name:        MetricPrefix + "/token_used_count",
+		Measure:     mTokenUsed,
+		Description: "The count of already used tokens on certificate issue",
+		TagKeys:     []tag.Key{observability.RealmTagKey},
+		Aggregation: view.Count(),
+	}); err != nil {
+		return nil, fmt.Errorf("stat view registration failure: %w", err)
+	}
+	mTokenInvalid := stats.Int64(MetricPrefix+"/invalid_token", "invalid tokens on certificate issue", stats.UnitDimensionless)
+	if err := view.Register(&view.View{
+		Name:        MetricPrefix + "/invalid_token_count",
+		Measure:     mTokenInvalid,
+		Description: "The count of invalid tokens on certificate issue",
+		TagKeys:     []tag.Key{observability.RealmTagKey},
+		Aggregation: view.Count(),
+	}); err != nil {
+		return nil, fmt.Errorf("stat view registration failure: %w", err)
+	}
+	mCertificateIssued := stats.Int64(MetricPrefix+"/issue", "certificates issued", stats.UnitDimensionless)
+	if err := view.Register(&view.View{
+		Name:        MetricPrefix + "/issue_count",
+		Measure:     mCertificateIssued,
+		Description: "The count of certificates issued",
+		TagKeys:     []tag.Key{observability.RealmTagKey},
+		Aggregation: view.Count(),
+	}); err != nil {
+		return nil, fmt.Errorf("stat view registration failure: %w", err)
+	}
+	mCertificateErrors := stats.Int64(MetricPrefix+"/errors", "certificate issue errors", stats.UnitDimensionless)
+	if err := view.Register(&view.View{
+		Name:        MetricPrefix + "/error_count",
+		Measure:     mCertificateErrors,
+		Description: "The count of certificate issue errors",
+		TagKeys:     []tag.Key{observability.RealmTagKey},
+		Aggregation: view.Count(),
+	}); err != nil {
+		return nil, fmt.Errorf("stat view registration failure: %w", err)
+	}
+
 	return &Controller{
-		config:      config,
-		db:          db,
-		h:           h,
-		logger:      logger,
-		pubKeyCache: pubKeyCache,
-		signerCache: signerCache,
-		kms:         kms,
+		config:             config,
+		db:                 db,
+		h:                  h,
+		logger:             logger,
+		pubKeyCache:        pubKeyCache,
+		signerCache:        signerCache,
+		kms:                kms,
+		mTokenExpired:      mTokenExpired,
```

Comment:
Can we create a metrics struct and function and pull the creation of it out of New into a metrics file?

kind of a nit